GatewaySDK/GatewaySDK/ViewController.swift

```diff
@@ -50,12 +50,12 @@ class ViewController: UIViewController {
             
             var RequestParams  = ZoopDataObject()
             RequestParams.zoop_env = "QT_PP"
-            RequestParams.zoop_gateway_id = "14a9ba38-4290-486f-8724-e4e2ee7c1cb6"
+            RequestParams.zoop_gateway_id = "5f86c1b22b0f3b0012b770d5"
```

Comment:
Why this is hard coded?

Reply:
this is just a sample app developer needs to call init api then they needs to generate and pass transaction id

Reply:
Okay